<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter un Equipement</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="eID"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Marque:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="e_marque"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <widget class="QComboBox" name="e_type">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Impremente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Clavier</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Souris</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Software</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Etat:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Fonctionnel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Defectieux</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>440</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Add_button">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Cancel_button">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>270</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>330</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>550</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AjouterLink">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>210</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Les Ã©quipements</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>631</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>351</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Equipement ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>40</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="tri_id">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Trier Par:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="tri_date_ajout">
         <property name="text">
          <string>Date d'ajout</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="tri_nom">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pdf_btn">
         <property name="text">
          <string>Generer PDF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="arduino">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>390</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Barcode Scanner</string>
      </property>
     </widget>
     <widget class="QPushButton" name="QR_btn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>470</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>QR Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="QRCODE_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>290</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Recherche</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Recherche">
         <property name="maximumSize">
          <size>
           <width>16777200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modifier un Equipement</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="id_comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="1">
        <widget class="QComboBox" name="e_type_2">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Impremente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Clavier</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Souris</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Software</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Marque:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="e_marque_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Etat:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Fonctionnel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Defectieux</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>420</y>
        <width>251</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="Update_button">
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Cancel_button_2">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Scanner Code A Barre</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>420</y>
        <width>251</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="addBarcode">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Cancel_button_4">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="res_arduino">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>351</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Equipement ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="eid"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
